src/components/Netcat/Netcat.tsx

```diff
@@ -215,78 +250,111 @@ const NetcatTool = () => {
+                    <NativeSelect
+                        value={selectedScanOption}
+                        onChange={(e) => setSelectedNetcatOption(e.target.value)}
+                        title={"Netcat option"}
+                        data={netcatOptions}
+                        required
+                        placeholder={"Pick a scan option"}
+                        description={"Type of scan to perform"}
+                    />
+                    {selectedScanOption === "Listen" && (
+                        <>
+                            <TextInput label={"Port number"} required {...form.getInputProps("portNumber")} />
+                        </>
+                    )}
+                    {selectedScanOption === "Connect" && (
+                        <>
+                            <TextInput label={"IP address"} required {...form.getInputProps("ipAddress")} />
+                            <TextInput label={"Port number"} required {...form.getInputProps("portNumber")} />
+                        </>
+                    )}
+                    {selectedScanOption === "Port Scan" && (
+                        <>
+                            <TextInput label={"IP address"} required {...form.getInputProps("ipAddress")} />
+                            <TextInput
+                                label={"Port number/Port range"}
+                                required
+                                {...form.getInputProps("portNumber")}
+                            />
+                        </>
+                    )}
+                    {selectedScanOption === "Send File" && (
+                        <>
+                            <TextInput label={"IP address"} required {...form.getInputProps("ipAddress")} />
+                            <TextInput label={"Port number"} required {...form.getInputProps("portNumber")} />
+                            <TextInput label={"File path"} required {...form.getInputProps("filePath")} />
+                        </>
+                    )}
+                    {selectedScanOption === "Receive File" && (
+                        <>
+                            <TextInput label={"Port number"} required {...form.getInputProps("portNumber")} />
+                            <TextInput label={"File path"} required {...form.getInputProps("filePath")} />
+                        </>
+                    )}
+                    {selectedScanOption === "Website Port Scan" && (
+                        <>
+                            <TextInput
+                                label={"Port number/Port range"}
+                                required
+                                {...form.getInputProps("portNumber")}
+                            />
+                            <TextInput label={"Domain name"} required {...form.getInputProps("websiteUrl")} />
+                        </>
+                    )}
+                    <Button type={"submit"}>start netcat</Button>
```

Comment:
Capital S for start and you can just use the variable for the title. 
```suggestion
                    <Button type={"submit"}>Start {Title}</Button>
```